<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListOfTripsAndCostsClass</class>
 <widget class="QWidget" name="ListOfTripsAndCostsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ListOfTripsAndCosts</string>
  </property>
  <widget class="QTableView" name="TripTableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>421</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="CostsTableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>421</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddTripPushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>180</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="EditCostID">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>390</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="EditCTripID">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>430</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="EditSum">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>470</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditCountry">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditCity">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="EditITripID">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QComboBox" name="SortComboBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>240</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Sort</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sort by country</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sort by departure date</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="EditDepartureDate">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>50</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="EditReturnDate">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>100</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteTripPushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>240</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UpgrateTripsPushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>180</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteCostPushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>600</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QComboBox" name="FilterComboBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>600</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Filter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Filter by description</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Filter by sum</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="AddCostPushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>530</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UpgrateCostsPushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>530</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DescriptionText">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>420</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Trips</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Expances</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Country</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>City</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Departure date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Return date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>430</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID of the trip</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>470</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sum</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>390</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="MinSum">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>520</y>
     <width>62</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="maxSum">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>560</y>
     <width>62</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="minlabel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>530</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxlabel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>560</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplyPushButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>600</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showAllTrips">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show all redords</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showAllCosts">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>610</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show all records</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelSortBtn">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>210</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel sorting</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ListOfTripsAndCosts.qrc"/>
 </resources>
 <connections/>
</ui>
